src/app/resource-actions/resource-actions.component.ts

```diff
@@ -0,0 +1,86 @@
+import { Component, OnInit } from '@angular/core';
+import {ProjectService} from '../../services/project-service/project.service';
+import {Resource} from '../../interfaces/resource';
+import {ResourceService} from '../../services/resource-service/resource.service';
+import {ActivatedRoute, Params} from '@angular/router';
+import {EntityActionsComponent} from '../entity-actions/entity-actions.component';
+import {ToastrService} from 'ngx-toastr';
+import {Model} from '../../interfaces/model';
+
+@Component({
+  selector: 'app-resource',
+  templateUrl: './resource-actions.component.html',
+  styleUrls: ['./resource-actions.component.css']
+})
+export class ResourceActionsComponent extends EntityActionsComponent implements OnInit {
+  public projectIndex: number;
```

Comment:
Why does it has an index? What if the server side returned the projects not by the same order each time? you shouldn't couple yourself to the order of the projects...